tests/browser/test/basic_block_test.js

```diff
@@ -37,14 +34,8 @@ suite('Basic block tests', function (done) {
 
   test('Drag three blocks into the workspace', async function () {
     for (let i = 1; i <= 3; i++) {
-      await dragBlockTypeFromFlyout(
-        browser,
-        'Basic',
-        'test_basic_empty',
-        250,
-        50 * i
-      );
-      chai.assert.equal((await getAllBlocks(browser)).length, i);
+      await dragNthBlockFromFlyout(browser, 'Align', 0, 250, 50 * i);
```

Comment:
Was there a reason this needed to be switched out?

Reply:
One of the basic blocks was breaking (I think it's one that you fixed in dev-tools) and it was making testing difficult.

I want to have a different test to validate that every test block can be shown in the toolbox and dragged in, but this particular test doesn't actually care which block you use.